### PR TITLE
Implemented eta xsi conversion

### PR DESCRIPTION
**Pull request by bernhardmgruber**
depends on #4

### CODE REVIEW

**Comment by rainman110** [comment]
Since we still have the cpacs2 structure in TiGL, I cannot verify the functionality.

I can still merge the code, but it would be either cool to have a switch (that the tools works also for the current TiGL). Or... we just update TiGL, which would make more sense to me.

@bernhardmgruber How much work would this be for you?

File: cpacs2to3/cpacs_converter.py [hunk 1]
```diff
@@ -387,19 +534,23 @@ def main():
...
 
+    # load old and new XML into TiGL
     tigl2 = tiglwrapper.Tigl()
     tigl2.open(old_cpacs_file, "")
 
     tigl3 = tigl3wrapper.Tigl3()
     tigl3.open(new_cpacs_file, "")
 
+    # perform non-structural conversions
+    convertEtaXsiValues(new_cpacs_file, tigl2, tigl3)
```
Comment (bernhardmgruber) [inline cpacs2to3/cpacs_converter.py:552]: if we comment out this line, the feature would be disabled for now

**Comment by bernhardmgruber** [comment]
updating TiGL to support the new eta and xsi iso lines probably takes some time. we have an internal prototype but it is in a super alpha state. we even believe that the CPACS schema is wrongly designed in some places (it changed several eta elements to eta iso lines, where the additional referenceUid does not make sense, e.g. etaStart/etaEnd in ribsPositioning which already has a member ribReference defining the eta reference line). i therefore cannot rule out the possibility of further schema corrections to be necessary.

long story short: TiGL3 should take its time with supporting eta xsi iso lines and we will have further discussions on this issue.

you can merge if you like or leave the PR open until TiGL is ready.

**Comment by rainman110** [comment]
Your code changes are integrated in commit b4dea737f840f80c
